worker/caasunitprovisioner/worker.go

```diff
@@ -180,11 +179,7 @@ func (p *provisioner) loop() error {
 					// not yet watching it and it's dead.
 					continue
 				}
-				cfg, err := p.config.ApplicationGetter.ApplicationConfig(appId)
-				if err != nil {
-					return errors.Trace(err)
-				}
-				jujuManagedUnits := cfg.GetBool(caas.JujuManagedUnits, false)
+				jujuManagedUnits := false // always set `jujuManagedUnits` to false
```

Comment:
I am not too sure why you would need to declare it if you are not going to use it.

Also, shouln't we have another one of these checks 'if this value is true, give user a warning and ignore'?